src/gpsea/preprocessing/_phenopacket.py

```diff
@@ -528,6 +534,10 @@ def _map_structural_type_to_variant_class(
                 return VariantClass.DEL
             elif structural_type.id in self._so_duplications:
                 return VariantClass.DUP
+            elif structural_type.id in self._so_translocations:
+                return VariantClass.TRANSLOCATION
```

Comment:
Depending on my comment in `_variant.py`, this may need to be changed to `VariantClass.BND`.

Reply:
I think BND is broader than just translocation ("BND can be used to represent reciprocal translocations or non-reciprocal break ends. ") but in any case, BND is a technical description rather than a biological one, and so I do not think we should actually keep BND as a Variant Category

Reply:
So do you recommend renaming `BND` to `TRANSLOCATION`?

Reply:
That would work, yes, I am guessing it has not been used in any other context yet (if at al!)!

Reply:
It has not been used, and I kind of planned to use `BND` as a translocation. Let's then keep `TRANSLOCATION` and remove `BND`.